package.json

```diff
@@ -21,11 +21,15 @@
         "build:compile": "babel -d lib --verbose --extensions \".ts,.tsx\" src",
         "start": "tsc -p ./tsconfig.build.json -w",
         "test": "jest",
-        "lint": "eslint src test && tsc --noEmit && prettier --check .",
-        "lint:fix": "prettier --write . && eslint --fix src test",
+        "lint": "yarn lint:js && yarn lint:workflows",
+        "lint:js": "eslint src test && tsc --noEmit && prettier --check .",
```

Comment:
not loving that we have a `lint:js` that does something different to the `lint:js` in element-web (notably: it runs `tsc`), but I also don't feel very strongly about it.